src/supla/storage/eeprom.cpp

```diff
@@ -56,20 +58,17 @@ int Eeprom::readStorage(unsigned int offset,
                         unsigned char *buf,
                         int size,
                         bool logs) {
-  if (logs) {
-    Serial.print(F("readStorage: "));
-    Serial.print(size);
-    Serial.print(F("; Read: ["));
-  }
+  int logSize = 0;
+  char logBuffer[4 * size];
```

Comment:
Please limit the "size" in some way. Allowing to put any size value from parameter may easly cause stack overflow. Use i.e. max 100 bytes.